cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -1254,5 +1263,72 @@ func NewUpstreamController(factory k8sinformer.SharedInformerFactory) (*Upstream
 	uc.ruleStatusChan = make(chan model.Message, config.Config.Buffer.UpdateNodeStatus)
 	uc.edgeClusterStateChan = make(chan model.Message, config.Config.Buffer.UpdateEdgeClusterState)
 	uc.missionStateChan = make(chan model.Message, config.Config.Buffer.UpdateMissionState)
+	uc.clusterGatewayChan = make(chan model.Message, config.Config.Buffer.UpdateClusterGateway)
 	return uc, nil
 }
+
+func (uc *UpstreamController) updateClusterGatewayNeighbors() {
+	for {
+		select {
+		case <-beehiveContext.Done():
+			klog.Warning("stop updateClusterGatewayNeighbors")
+			return
+		case msg := <-uc.clusterGatewayChan:
+			klog.V(5).Infof("message: %s, operation is: %s, and resource is %s", msg.GetID(), msg.GetOperation(), msg.GetResource())
```

Comment:
probably remove "message:" in such logs. it doesn't provide additional info.

Reply:
Removed